===================================================================
RCS file: /horde/cvs/passwd/main.php,v
retrieving revision 1.1.1.1.2.11
retrieving revision 1.1.1.1.2.12
diff -p --unified=3 -r1.1.1.1.2.11 -r1.1.1.1.2.12
--- main.php	2004/03/26 22:43:28	1.1.1.1.2.11
+++ main.php	2004/06/05 21:40:52	1.1.1.1.2.12
@@ -137,7 +137,7 @@ do {
     }
     if (array_key_exists('maxSpace', $passwordPolicy) &&
         $passwordPolicy['maxSpace'] < $space) {
-        if ($paswordPolicy['maxSpace'] > 0) {
+        if ($passwordPolicy['maxSpace'] > 0) {
             Horde::raiseMessage(sprintf(_("Your new password must contain less than %d whitespace characters."), $passwordPolicy['maxSpace']), HORDE_WARNING);
         } else {
             Horde::raiseMessage(_("Your new password must not contain whitespace characters."), HORDE_WARNING);
